src/scss/components/_about.scss

```
@@ -178,42 +136,34 @@
 
     &__link {
       @extend %text-style-ui-3;
-      color: $color-brand-blue-600;
+      color: $color-light-yellow;
 
       .back-top__icon-container {
-        // color brand-blue-600
-        filter: invert(39%) sepia(72%) saturate(6596%) hue-rotate(200deg)
-          brightness(100%) contrast(84%);
+        fill: $color-light-yellow;
         transition: $transition__link;
       }
 
       &:hover {
-        color: $color-brand-blue-700;
+        color: $color-dark-yellow;
+        transition: $transition__link;
 
         .back-top__icon-container {
-          // color: $color-brand-blue-700;
-          filter: invert(22%) sepia(54%) saturate(2858%) hue-rotate(194deg)
-            brightness(92%) contrast(101%);
+          fill: $color-dark-yellow;
           transition: $transition__link;
         }
       }
 
       &:focus {
-        color: $color-brand-blue-700;
-        background-color: $color-background-light-blue-100;
+        color: $color-main-red;
+        background-color: $color-off-white;
         outline: 0;
 
         .back-top__icon-container {
-          // color: $color-brand-blue-700;
           filter: invert(22%) sepia(54%) saturate(2858%) hue-rotate(194deg)
             brightness(92%) contrast(101%);
           transition: $transition__link;
         }
       }
```

For accessibility and to make it easy for the user to know which links already visited. we need to add the visited pseudo class
```suggestion
      }
      &:visited {
        color: $color-brand-blue-870; //update it with the color you need
      }
```